src/ashtech.h

```diff
@@ -72,16 +72,16 @@ class GPSDriverAshtech : public GPSHelper
 		NME_DECODE_GOT_FIRST_CS_BYTE
 	};
 
-	struct satellite_info_s *_satellite_info;
-	struct vehicle_gps_position_s *_gps_position;
-	uint64_t _last_timestamp_time;
-	int _ashtechlog_fd;
+	struct satellite_info_s *_satellite_info {nullptr};
```

Comment:
It sounds like uniform initialization is the way forward. http://www.stroustrup.com/C++11FAQ.html#uniform-init
What do you think?

Perhaps we should establish some basic coding standards to gradually bring more consistency?